Build failure vtkParallelTransportTest1.cxx

Fix regression introduced in https://github.com/Slicer/vtkAddon/commit/cf1126568415179fb5b1c16121087bd2494e2e0e

```
vtkAddon/Testing/vtkParallelTransportTest1.cxx:108:7: error: ‘tangentsArray’ was not declared in this scope; did you mean ‘tangents’?
```

Source: https://slicer.cdash.org/viewBuildError.php?buildid=4005586

---

Related pull request:
* https://github.com/Slicer/vtkAddon/pull/59

--- a/Testing/vtkParallelTransportTest1.cxx
+++ b/Testing/vtkParallelTransportTest1.cxx
@@ -96,6 +96,7 @@ int vtkParallelTransportTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)[])
     vtkPointData* pointData = parallelTransportFrame->GetOutput()->GetPointData();
     vtkDoubleArray* normalsArray = vtkDoubleArray::SafeDownCast(pointData->GetArray("Normals"));
     vtkDoubleArray* binormalsArray = vtkDoubleArray::SafeDownCast(pointData->GetArray("Binormals"));
+    vtkDoubleArray* tangentsArray = vtkDoubleArray::SafeDownCast(pointData->GetArray("Tangents"));
     double constantBinormal[3] = { 0.0, 0.0, 0.0 };
     binormalsArray->GetTypedTuple(0, constantBinormal);
     for (vtkIdType tupleIndex = 1; tupleIndex < normalsArray->GetNumberOfTuples(); tupleIndex++)

